helper/logging/logging.go

```diff
@@ -114,3 +138,9 @@ func isValidLogLevel(level string) bool {
 
 	return false
 }
+
+func GetTestLogContext(t testing.T) context.Context {
```

Comment:
I wonder if we should mark this one as not following our normal compatibility promises:

```suggestion
// GetTestLogContext creates a context that is registered to the SDK log sink.
// This function is for internal usage only and is not supported by the project's
// compatibility promises.
func GetTestLogContext(t testing.T) context.Context {
```